src/platform/src/common/components/Modal/dataDownload/modules/DataDownload.jsx

```
@@ -322,14 +404,206 @@ const DataDownload = ({ onClose }) => {
+      // Update the active filter key
+      setActiveFilterKey(activeFilter.key);
+
+      // Return data based on filter type
+      switch (activeFilter.key) {
+        case 'countries':
+          return countriesData || [];
+        case 'cities':
+          return citiesData || [];
+        case 'devices':
+          return devicesData || [];
+        case 'sites':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)